findex_sqlite/src/error.rs

```diff
@@ -0,0 +1,53 @@
+use std::fmt::Display;
+
+#[derive(Debug)]
+#[allow(clippy::enum_variant_names)]
```

Comment:
Could we comply to clippy here instead of ignoring? It's not a big change?

Reply:
Could you explain yourself ?

Reply:
Clippy is complaining that their is an `Error` suffix, no? Could we remove the `Error` suffix inside the enum instead of `#[allow(clippy::enum_variant_names)]`?

Reply:
I removed it, clippy doesn't seem to comply anymore...

Reply:
In eaf2174